swig/CMakeLists.txt

```diff
@@ -109,6 +109,19 @@ install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/cpp/src/" DESTINATION ${CMAKE_INS
 # Examples
 if(GEN_CPP_BINDINGS AND BUILD_CPP_EXAMPLES)
 	add_subdirectory(cpp/examples)
+  add_subdirectory(cpp/tests)
+
+  set(CPP_TEST_MODULE "swig-test")
+  set(CPP_TEST_MODULE_REV "2017-03-09")
+  set(CPP_SWIG_DIR ${CMAKE_CURRENT_SOURCE_DIR}/cpp)
+  set(CPP_TEST_DIR ${CPP_SWIG_DIR}/tests)
+  set(CPP_YANG_DIR ${CPP_TEST_DIR}/yang)
+
+  ADD_CUSTOM_COMMAND(
+    TARGET Sysrepo-cpp
+    POST_BUILD
+    COMMAND ${CMAKE_BINARY_DIR}/src/sysrepoctl --install --yang=${CPP_YANG_DIR}/${CPP_TEST_MODULE}@${CPP_TEST_MODULE_REV}.yang
```

Comment:
This looks like being in conflict with the change I introduced in 00025c4. Please ensure that it only runs when tests are enabled, otherwise the build will start failing again during cross-compilation.